PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/ui/SparkClusterListRefreshableCombo.kt

```diff
@@ -107,7 +111,7 @@ open class SparkClusterListRefreshableCombo: ILogger, Disposable {
     open inner class ViewModel(private val initClusters: Array<IClusterDetail>,
                                private val clusterIdMapper: (IClusterDetail?) -> String? = { cluster -> cluster?.name })
         : DisposableObservers() {
-
+        var preSelectClusterName :String? = null
```

Comment:
Unused var?

Reply:
used to record the preselectcluster in the combox deselect event

---
In reply to: [243788406](https://github.com/Microsoft/azure-tools-for-java/pull/2520#discussion_r243788406) [](ancestors = 243788406)